### ISSUE TITLE
 losses appear to be 0

### ISSUE THREAD

**Issue by 1YuChenyao** [OpenDriveLab/UniAD#194]
The learning rate has been reduced to 1e-4, and 2 pictures were run at  4090. Why do some losses appear to be 0 during the second phase of training?
![image](https://github.com/user-attachments/assets/2b2377a9-8b26-4768-a20a-052282c87be4)

**Comment by OrangeSodahub** [comment]
@1YuChenyao Yes, I have the same question as you. Did you figure it out? In my case, the `planning.loss_collision` is zero!

**Comment by 1YuChenyao** [comment]
Sorry，I am not solve this question

---- Replied Message ----
| From | ***@***.***> |
| Date | 08/22/2024 11:54 |
| To | OpenDriveLab/UniAD ***@***.***> |
| Cc | ycy ***@***.***>,
Mention ***@***.***> |
| Subject | Re: [OpenDriveLab/UniAD] losses appear to be 0 (Issue #194) |

@1YuChenyao Yes, I have the same question as you. Did you figure it out? In my case, the planning.loss_collision is zero!
